src/pages/InsideContents/settings/MyQuotes.tsx

```
@@ -0,0 +1,235 @@
+                        >
+                          <DeleteIcon fontSize="small" />
+                        </IconButton>
+                      </Box>
+                    </Box>
+                    <Typography variant="body2" color="textSecondary" sx={{ mb: 2 }}>
+                      {quote.requirements?.materials || 'No description available'}
+                      {quote.size?.dimensions && ` - ${quote.size.dimensions}`}
+                    </Typography>
+                    <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+                      <Typography variant="h6" color="primary">
+                        {(quote as any).fixedEstimate || 'Price not available'}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)